### PR TITLE
Show anchor and origin in skin blueprints when selected

### PR DESCRIPTION
**Pull request by peppy**
- [x] Depends on #12762.

20210512 151131 (dotnet)

### CODE REVIEW

**Comment by frenzibyte** [comment]
FWIW skinnable components with applied margin/padding will make the origin visualiser floating off, it sounds like `DefaultHealthDisplay` shouldn't have any margin applied but have a defined offset `Position` instead (on the default skin components json), but just gonna throw a note here for now.

CleanShot 2021-05-12 at 21 27 03

(notice `OriginPosition = (0, -20)`)

**Comment by peppy** [comment]
It's actually showing correctly in this case (the anchor point *should* include the margin/padding). Whether that margin/padding should be removed or not is for a separate discussion. If we keep such definitions - which I think we may want to for flowing layouts - it will also need to be visualised.

File: osu.Game/Skinning/Editor/SkinBlueprint.cs [hunk 1]
```diff
@@ -118,4 +127,58 @@ protected override void Update()
...
+        {
+            this.drawable = drawable;
+
+            InternalChildren = new Drawable[]
+            {
+                anchorLine = new Box
+                {
+                    Colour = Color4.Yellow,
+                    Height = 2,
+                },
```
Comment (bdach) [inline osu.Game/Skinning/Editor/SkinBlueprint.cs:150]: Not 100% on why this is not a path (would save on the atan math) but I guess it does seem to work for now.

**Review by bdach — APPROVED** [review]

**Comment by peppy** [comment]
I tried using a path first and it's a bit weird (you need to position the path weirdly in the case of negative sizing).

**Pull request merged**